packages/editor-ui/src/typeGuards.ts

```diff
@@ -7,3 +8,7 @@ export function isResourceLocatorValue(value: unknown): value is INodeParameterR
 export function isNotNull<T>(value: T | null): value is T {
 	return value !== null;
 }
+
+export function isValidCredentialResponse(value: {}): value is ICredentialsResponse {
```

Comment:
Just a nitpick: you should not use the object literal as type, nor `object` the most proper way would be `Record<string, any>` (or instead of `any` the proper type if you know them)

Reply:
Agree, updating.